integration_tests/tests/flowmachine_tests/conftest.py

```diff
@@ -0,0 +1,18 @@
+import pandas as pd
+import pytest
+
+from approvaltests.reporters.generic_diff_reporter_factory import (
+    GenericDiffReporterFactory,
+)
+
+
+@pytest.fixture
+def get_dataframe(fm_conn):
+    yield lambda query: pd.read_sql_query(query.get_query(), con=fm_conn.engine)
+
+
+@pytest.fixture(scope="session")
+def diff_reporter():
+    diff_reporter_factory = GenericDiffReporterFactory()
+    return diff_reporter_factory.get("opendiff")
+    # return diff_reporter_factory.get_first_working()
```

Comment:
Has this comment been left intentionally?

Reply:
Hmm, initially yes but it's probably better to remove it for now.